src/components/display/Table/components/PercentageIndicator/index.tsx

```diff
@@ -3,14 +3,14 @@ import React from "react";
 import NumberFormat from "react-number-format";
 
 // Assets
-import ArrowChartUpIcon from "@assets/icons/arrow-chart-up-icon.svg";
-import ArrowChartDownIcon from "@assets/icons/arrow-chart-down-icon.svg";
+import ArrowChartUpIcon from "../../../../../assets/icons/arrow-chart-up-icon.svg";
+import ArrowChartDownIcon from "../../../../../assets/icons/arrow-chart-down-icon.svg";
```

Comment:
Why do you need these relative imports? It looks really strange with "../../../". Can we make them absolute paths instead?

Reply:
At first we had somewhat unfinished relative routes.

On one side @assets/*, and on the other assets/*. Then in the new refactor we started using assets/*.

If you want I'll check the project and add absolute paths to the elements.